org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/parser/antlr/AntlrToolFacade.java

```diff
@@ -122,13 +127,34 @@ public boolean download() {
 			}
 			log.info("Finished downloading.");
 		} catch (IOException e) {
-			System.err.println("Downloading ANTLR parser generator failed: " + e.getMessage());
-			System.err.println("Please install the feature 'Xtext Antlr SDK' manually using the external updatesite:");
-			System.err.println();
-			System.err.println("\t\t'http://download.itemis.com/updates/'."); 
-			System.err.println();
-			System.err.println("(see http://www.eclipse.org/Xtext/download.html for details)");
-			return false;
+			try {
+				log.warn("Downloading file from '"+secureDownloadURL+"' failed.", e);
```

Comment:
should this become an error instead?

Reply:
should we log the exception or skip it/log the message only?

Reply:
I think it's fine.